app/metrics/admin.py

```diff
@@ -54,22 +57,70 @@ def save_model(self, request, obj, form, change):
 
 @admin.register(QuestionSet)
 class QuestionSetAdmin(ModelAdmin):
-    exclude = EDIT_TRACKING_FIELDS
     prepopulated_fields = {"slug": ["name"]}
 
     list_display = (
         "name",
         "user",
     )
 
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        if request.user.is_superuser:
+            return qs
+        
+        user_node = request.user.get_node()
+        if user_node:
+            # Filter QuestionSets based on the node
+            exclude = EDIT_TRACKING_FIELDS  # needs to be included if we want to be able to change user for a set
```

Comment:
I don't understand how this would make any difference. Nothing is referencing `exclude` in this scope. Right?